common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -206,3 +206,37 @@ class VideoFields(object):
         scope=Scope.preferences,
         default=False,
     )
+
+    enable_progress_bar = Boolean(
+        help=_(
+            "Specify whether progress bar is enabled or disabled"
+        ),
+        display_name=_("Enable Progress Bar"),
+        scope=Scope.settings,
+        default=False
+    )
+
+    enable_speed_control = Boolean(
+        help=_(
+            "Specify whether Speed Controls are enabled or disabled"
+        ),
+        display_name=_("Enable Speed Controls"),
+        scope=Scope.settings,
+        default=False
+    )
+
+    enable_next_button = Boolean(
+        help=_(
+            "Specify whether next button in LMS is enabled or disabled"
```

Comment:
change to `Specify whether next button in LMS is enabled when video is marked completed`